kibit/src/kibit/rules/collections.clj

```diff
@@ -20,5 +20,7 @@
   ;; set
   [(into #{} ?coll) (set ?coll)]
 
-  [(take ?n (repeatedly ?coll)) (repeatedly ?n ?coll)])
+  [(take ?n (repeatedly ?coll)) (repeatedly ?n ?coll)]
+  [(doall (map ?fn ?coll)) (run! ?fn ?coll)]
+  [(doall (mapv ?fn ?coll)) (run! ?fn ?coll)])
```

Comment:
`mapv` isn't lazy and is frequently used as a strict alternative to `map` which still participates in the sequence protocol. This substitution isn't needed.